coding/browser-support.md

```diff
@@ -0,0 +1,31 @@
+# Browser support
+
+Our baseline browser support for brochure style websites. There are two levels
```

Comment:
Why the "brochure style" distinction?

Reply:
Because the apps have different browser support.

Reply:
Hmm. Maybe we could say this is for a specific team then? Or say "unless there are specific reasons for diverging"? Or something?